src/css/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  background-color: var(--white-color);
+  border: var(--card-border);
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  display: block;
+  margin: 0 3px 40px;
+}
+
+.card__title {
+  color: var(--primary-color);
+  font-family: inherit;
+  font-size: var(--font-size-sm);
+  line-height: var(--line-height-lg);
+  font-weight: var(--font-weight-medium);
+  margin: 0 0 4px;
+}
+
+.card__prodcode {
+  color: var(--secondary-color);
+  font-family: inherit;
+  font-size: var(--font-size-xs);
+  line-height: var(--line-height-sm);
+  font-weight: var(--font-weight-light);
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__rating-reviews {
+  color: var(--primary-color);
+  font-family: inherit;
+  font-weight: var(--font-weight-light);
+  font-size: var(--font-size-xs);
+  line-height: var(--line-height-sm);
+}
+
+.card__price-label {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-text {
+  color: var(--secondary-color);
+  font-family: inherit;
+  font-size: var(--font-size-sm);
+  line-height: var(--line-height-lg);
+  font-weight: var(--font-weight-light);
+}
+
+.card__price-number {
+  color: var(--primary-color);
+  font-family: inherit;
+  font-size: var(--font-size-lg);
+  line-height: var(--line-height-lg);
+  font-weight: var(--font-weight-bold);
+}
+
+.card__btn {
+  display: block;
+  text-transform: uppercase;
+  color: var(--white-color);
+  font-family: inherit;
+  text-decoration: none;
+  text-align: center;
+  font-size: var(--font-size-md);
+  font-weight: var(--font-weight-bold);
+  background-color: var(--theme-color);
+  line-height: var(--line-height-md);
+  padding: 11px 0;
+  border-radius: 5px;
+  transition: 0.3s all ease;
+  border: 1px solid var(--theme-color);
+}
+
+.card__btn:hover {
+  color: var(--theme-color);
+  background-color: var(--white-color);
+  border: 1px solid var(--theme-color);
```

Comment:
You already have a border om 91 line

Reply:
Thank you for review! There is no reason to answer for all of remarks now. I can see some of them are results of my personal preferences, but there are couple real mistakes and these errors I'll fix. Good luck!

Reply:
Thank you too!